jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -109,7 +109,7 @@ ecma_op_is_callable (ecma_value_t value) /**< ecma value */
  * @return true - if the given object is constructor;
  *         false - otherwise
  */
-inline bool JERRY_ATTR_ALWAYS_INLINE
+inline bool
```

Comment:
Why did you removed this?

Reply:
**ecma-function-object.c:113:1**: error: inlining failed in call to always_inline ‘ecma_object_is_constructor’: function not considered for inlining

Function has recursive call to itself - i think that's why compilator failed to inline it. But I left first *inline* untouched - it still can be used as suggestion to compilator when building with optimization enabled.

Reply:
Ok, I see it now. What about using a while loop to skip bound functions? This way the recursion can be removed (saves stack).

Reply:
Done.